### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by lakshmana-git**
week1

### CODE REVIEW

File: Week-1/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+# Week 1 Assignemnt 
+
+Create a web page with HTML and CSS
+
+font used:Lato
+
```
Comment (ShrutikaSingh) [inline Week-1/README.md:7]: why empty line

File: Week-1/Week-1 [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+My WhiteBoard feature enable teacher to explain the concepts in more efficient manner in this digital era.Students who are listening to the class can now save there boards and revise wherever they need.
+
+As the world is fighting with pandemic and staying at home is more pivotal in this pandemic situation.But due this the  progress and improvements had stopped .
+As we know teaching and learning should never stop because  thats the base line for problem solving and development.
+
+My whiteboard solves the problem of making teaching is more easy then ever before.Teacher now can teach the students in more systamtic manner from there homes.My whiteboards also let students to connect with the enivronment with interactive features.
+
+My whiteboard has unique feature of voice assistant  (simba) that helps to wrtie on board what the teacher is talking about.The teacher shouldnt need to open a new tab to seach for anything,My whiteboard has feature to search anything and that will be displayed on board with split screen or full screen.My Whiteboard can store the class data in each class folder and retrive previous board if required.
+
```
Comment (ShrutikaSingh) [inline Week-1/Week-1:15]: remove extra empty line and do proper identention

File: Week-1/assignment1_1.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+If OS also dont have ,Browser looks up and makes a query out to internet  to a DNS resolver.This is set off a chain of requests untill the IP address is resolved
+
+
+Now the browser has IP addresss
+
+browser establish a TCP connection with the server  Web server
+
+browser send HTTp request to server
+
+server sends back HTTP reponse
```
Comment (ShrutikaSingh) [inline Week-1/assignment1_1.md:23]: server send back http response with some http codes like 200, 404, etc mention those and read about http codes

File: Week-1/assignment1_1.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+
+
+What is the main functionality of the browser?
+The main function of a browser is to present the web resource you choose, by requesting it from the server and displaying it in the browser window. The resource is usually an HTML document, but may also be a PDF, image, or some other type of content.
+
+A web browser takes you anywhere on the internet. It retrieves information from other parts of the web and displays it on your desktop or mobile device. The information is transferred using the Hypertext Transfer Protocol, which defines how text, images and video are transmitted on the web.
+
+
+High Level Components of a browser
```
Comment (ShrutikaSingh) [inline Week-1/assignment1_1.md:35]: good answer

File: Week-1/assignment1_1.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+A render knows how to lay out and paint itself and its children.
+
+
+
+Order of script processing
+
+
+Layout and Painting
+imultaneously, the browser also creates a render tree. This tree includes both the styling information as well as the visual instructions that define the order in which the elements will be displayed. The render tree ensures that the content is displayed in the desired order.
```
Comment (ShrutikaSingh) [inline Week-1/assignment1_1.md:105]: imultaneously?

File: Week-1/index.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+            <div class="header-inside">
+              <div>
+                <h1>CattyBoard Top 100 Single Charts(11.06.36)</h1>
+                <p>Unknown Artist</p>
+                <p>2016.Charts.100 songs</p>
+              </div>
+              <div class="header-down">
+                <div>
+                  <p>
+                    <i class="fa fa-play" aria-hidden="true"></i>&nbsp Play all
```
Comment (ShrutikaSingh) [inline Week-1/index.html:46]: &nbsp is used with ; like &nbsp for space, try not to use &nbsp; and manage the space with css property like padding, margin, etc

File: Week-1/index.html [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+                  </p>
+                </div>
+                <div>
+                  <p>
+                    <i class="fa fa-plus" aria-hidden="true"></i> &nbspAdd to
```
Comment (ShrutikaSingh) [inline Week-1/index.html:51]: +1

File: Week-1/index.html [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+                  </p>
+                </div>
+                <div>
+                  <p>
+                    <i class="fa fa-ellipsis-h" aria-hidden="true"></i>
+                    &nbspMore
```
Comment (ShrutikaSingh) [inline Week-1/index.html:57]: +1

File: Week-1/index.html [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+        <div class="footer-name">
+          <p class="footer-text">If It Ain't Love</p>
+          <p>Caston Derulo</p>
+        </div>
+      </div>
+      <div class="footer-icon">
+        <i class="fa fa-step-backward" aria-hidden="true"></i>
+        <i class="fa fa-pause" aria-hidden="true"></i>
+        <i class="fa fa-step-forward" aria-hidden="true"></i>
+        <i class="fa fa-undo" aria-hidden=""></i>
```
Comment (ShrutikaSingh) [inline Week-1/index.html:112]: if no value is given to aria-hidden="", donot use it

File: Week-1/style.css [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+    margin-bottom: 15px;
+  }
+  .footer-icon {
+    display: flex;
+    align-self: center;
+    color: white;
+    width: 600px;
+    justify-content: space-between;
+  }
+
```
Comment (ShrutikaSingh) [inline Week-1/style.css:105]: add empty line at the end of file

File: Week-2/index2-1.html [hunk 11]
```diff
@@ -0,0 +1,359 @@
+<!DOCTYPE html>
```
Comment (ShrutikaSingh) [inline Week-2/index2-1.html:1]: donot push week1 and week 2 codes together, this time i am reviewing it, from next time you should take care

File: Week-2/index2-1.html [hunk 12]
```diff
@@ -0,0 +1,359 @@
...
+              </label>
+            
+            </div>
+           <div class="form-div">
+            <label>Name</label>
+            <input type="text" required>
+           </div>
+           <div class="form-div">
+            <label>Last Name</label>
+            <input type="text" required>
```
Comment (ShrutikaSingh) [inline Week-2/index2-1.html:63]: some fileds might not be required, depends on the use case

File: Week-2/index2-2.html [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Static Template</title>
+    <link rel="stylesheet" href="style2-2.css" />
+  </head>
+  <body>
+    <div class="card">
+      <div class="main">
+        <div class="card1"></div>
```
Comment (ShrutikaSingh) [inline Week-2/index2-2.html:13]: why empty div?

File: Week-2/style2-1.css [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+    font-size: 4rem;
+    border-style:solid;
+    border-width:5px;
+    
+}
+img{
+    height:17rem;
+    width:40rem
+
+
```
Comment (ShrutikaSingh) [inline Week-2/style2-1.css:29]: remove empty  lines

File: Week-2/style2-2.css [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+  }
+  
+  .main:hover .card3 {
+    transform: rotate(-90deg);
+  }
+  
+  .main:hover .card4 {
+    transform: rotate(90deg);
+  }
+
```
Comment (ShrutikaSingh) [inline Week-2/style2-2.css:76]: add empty line

File: Week-1/README.md (hunk 1 above)
Comment (lakshmana-git) [reply Week-1/README.md:7]: I will update

File: Week-1/assignment1_1.md (hunk 3 above)
Comment (lakshmana-git) [reply Week-1/assignment1_1.md:23]: I will learn mam

File: Week-1/assignment1_1.md (hunk 4 above)
Comment (lakshmana-git) [reply Week-1/assignment1_1.md:35]: Thankyou mam

File: Week-1/assignment1_1.md (hunk 5 above)
Comment (lakshmana-git) [reply Week-1/assignment1_1.md:105]: wrong type mam ,It is simultaneously .I updated it mam

File: Week-1/index.html (hunk 6 above)
Comment (lakshmana-git) [reply Week-1/index.html:46]: Sure mam

File: Week-1/index.html (hunk 7 above)
Comment (lakshmana-git) [reply Week-1/index.html:51]: sure mam

File: Week-2/index2-1.html (hunk 11 above)
Comment (lakshmana-git) [reply Week-2/index2-1.html:1]: ok mam

File: Week-2/index2-2.html (hunk 13 above)
Comment (lakshmana-git) [reply Week-2/index2-2.html:13]: To make understanding better,I added an empty div mam

File: Week-1/index.html (hunk 9 above)
Comment (lakshmana-git) [reply Week-1/index.html:112]: ok mam

File: Week-2/index2-1.html (hunk 12 above)
Comment (lakshmana-git) [reply Week-2/index2-1.html:63]: ok mam i will check

**Pull request merged**